### PR TITLE
Missing ItemType Methods

### PR DESCRIPTION
**Pull request by Codinablack**
-added some new itemType methods
-altered isRune boolean in items.h for consistency

### CODE REVIEW

**Comment by Codinablack** [comment]
This list 

https://github.com/otland/forgottenserver/issues/2293

File: src/luascript.h [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/**
+﻿/**
```
Comment (ranisalt) [inline src/luascript.h]: What happened here? I can't find any diff.

**Comment by ranisalt** [comment]
Thanks!

File: src/luascript.h (hunk 1 above)
Comment (ranisalt) [inline src/luascript.h]: It's git drunk again.

**Pull request merged**